pkg/driver/node_test.go

```diff
@@ -1161,73 +1225,92 @@ func TestGetVolumesLimit(t *testing.T) {
 			metadataMock: func(ctrl *gomock.Controller) *metadata.MockMetadataService {
 				m := metadata.NewMockMetadataService(ctrl)
 				m.EXPECT().GetRegion().Return("us-west-2")
-				m.EXPECT().GetInstanceType().Return("mac1.metal")
+				m.EXPECT().GetInstanceType().Return("dl1.24xlarge")
 				m.EXPECT().GetNumBlockDeviceMappings().Return(0)
 				m.EXPECT().GetNumAttachedENIs().Return(1)
 				return m
 			},
 		},
 		{
-			name: "u-12tb1.metal_volume_attach_limit",
+			// will and should fail if g5.48xlarge instance type is in any table other than maxVolumeLimits table
```

Comment:
Hyper-nit: Add space between `limit` and `(` like we do for other unit tests

Also consider rephrasing parenthetical to `(Instance has attached GPUs and NVMe Instance Store volumes but should be ignored for EBS volume limits calculation)` to clarify the different devices/limits at play here for future contributors. 

Same thing for rest of new unit tests.